### PR TITLE
Implemented CRUD in Windows form

### PR DESCRIPTION
**Pull request by hassan22khan**

### CODE REVIEW

File: Internship Project/Auth/scripts/studentManager.js [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    })">Delete</button></td>
+        </tr>`
+  );
+  $("#student-records").append(table);
+}
+
+function logOut(){
+  sessionStorage.removeItem("accessToken");
+  sessionStorage.removeItem("dataPlusToken");
+  location.href = "http://127.0.0.1:5500/logUser.html";
```
Comment (M-Jawad5669) [inline Internship Project/Auth/scripts/studentManager.js:330]: Use dynamic urls, static urls are only feasible to your system.
example: static url: "http://127.0.0.1:5500/logUser.html"
          dynamic url: window.location.origin + "/logUser.html" etc

File: Internship Project/InternWebApi/Controllers/StudentController.cs [hunk 2]
```diff
@@ -21,10 +26,25 @@ public IHttpActionResult GetStudents(string id)
             return Ok(_repo.GetStudentCourses(students));
```
Comment (M-Jawad5669) [inline Internship Project/InternWebApi/Controllers/StudentController.cs:26]: For repositories use proper name like _studentRepository or whatever, using only  _repo is not so meaningful. we may have multiple repositories in the same controller.

File: Internship Project/Auth/scripts/studentManager.js (hunk 1 above)
Comment (hassan22khan) [reply Internship Project/Auth/scripts/studentManager.js:330]: Thanks that is a helpful tip . Will do it now

File: Internship Project/InternWebApi/Controllers/StudentController.cs (hunk 2 above)
Comment (hassan22khan) [reply Internship Project/InternWebApi/Controllers/StudentController.cs:26]: Yeah ok will do it

**Pull request merged**